### ISSUE TITLE
Modify DataServer to use plate-model-manager

### ISSUE THREAD

**Issue by brmather** [GPlates/gplately#135]
We should encourage people to use the `plate-model-manager`, as @michaelchin is doing for the notebooks, but we should also adapt the existing `DataServer` object to use `plate-model-manager` under the hood so that we don't upset existing GPlately users. We can always deprecate `DataServer` in the future if we need to.

@michaelchin - I'm not sure if this is already an option, but can `plate-model-manager` download files to the user's cache directory?

**Comment by michaelchin** [comment]
> @michaelchin - I'm not sure if this is already an option, but can `plate-model-manager` download files to the user's cache directory?

Yes, the `plate-model-manager` can save the files to user-specified folder.

**Comment by michaelchin** [comment]
> We should encourage people to use the `plate-model-manager`, as @michaelchin is doing for the notebooks, but we should also adapt the existing `DataServer` object to use `plate-model-manager` under the hood so that we don't upset existing GPlately users. We can always deprecate `DataServer` in the future if we need to.

Yes, this is a good idea. Another option is to keep both for a while. The `plate-model-manager` is new and maybe have some bugs. If we keep `DataServer` as it is, users have the option to fall back to `DataServer` if the `plate-model-manager` does not work for them.

**Comment by michaelchin** [comment]
Maybe print deprecated message when users use `DataServer ` to encourage them to switch to the new `plate-model-manager`.

**Comment by michaelchin** [comment]
Hi @brmather 

Are you still on this? I can finish this if you like.

**Comment by michaelchin** [comment]
this issue is related to #149
